akka-cluster-sharding-typed/src/test/scala/docs/akka/cluster/sharding/typed/AccountExampleWithCommandHandlersInState.scala

```diff
@@ -24,43 +24,41 @@ object AccountExampleWithCommandHandlersInState {
   //#account-entity
   object AccountEntity {
     // Command
-    sealed trait AccountCommand[Reply] extends ExpectingReply[Reply] with CborSerializable
-    final case class CreateAccount(override val replyTo: ActorRef[OperationResult])
-        extends AccountCommand[OperationResult]
+    sealed trait Command[Reply <: CommandReply] extends ExpectingReply[Reply] with CborSerializable
+    final case class CreateAccount(override val replyTo: ActorRef[OperationResult]) extends Command[OperationResult]
     final case class Deposit(amount: BigDecimal, override val replyTo: ActorRef[OperationResult])
-        extends AccountCommand[OperationResult]
+        extends Command[OperationResult]
     final case class Withdraw(amount: BigDecimal, override val replyTo: ActorRef[OperationResult])
-        extends AccountCommand[OperationResult]
-    final case class GetBalance(override val replyTo: ActorRef[CurrentBalance]) extends AccountCommand[CurrentBalance]
-    final case class CloseAccount(override val replyTo: ActorRef[OperationResult])
-        extends AccountCommand[OperationResult]
+        extends Command[OperationResult]
+    final case class GetBalance(override val replyTo: ActorRef[CurrentBalance]) extends Command[CurrentBalance]
+    final case class CloseAccount(override val replyTo: ActorRef[OperationResult]) extends Command[OperationResult]
 
     // Reply
-    sealed trait AccountCommandReply extends CborSerializable
-    sealed trait OperationResult extends AccountCommandReply
+    sealed trait CommandReply extends CborSerializable
+    sealed trait OperationResult extends CommandReply
     case object Confirmed extends OperationResult
     final case class Rejected(reason: String) extends OperationResult
-    final case class CurrentBalance(balance: BigDecimal) extends AccountCommandReply
+    final case class CurrentBalance(balance: BigDecimal) extends CommandReply
 
     // Event
-    sealed trait AccountEvent extends CborSerializable
-    case object AccountCreated extends AccountEvent
-    case class Deposited(amount: BigDecimal) extends AccountEvent
-    case class Withdrawn(amount: BigDecimal) extends AccountEvent
-    case object AccountClosed extends AccountEvent
+    sealed trait Event extends CborSerializable
```

Comment:
I don't know that we want to suggest in style docs that users name a base event `Event`

Reply:
That's the convention we should use, similar to that `Command` is not prefixed. When used from the outside it becomes `AccountEntity.Command` and `AccountEntity.Event`.